apps/frontend/src/pages/management/libs/components/users-tab/users-tab.tsx

```diff
@@ -104,8 +126,11 @@ const UsersTab: React.FC = () => {
 						className={styles["icon-button"]}
 						hasVisuallyHiddenLabel
 						iconName="delete"
-						isDisabled={!hasPermissionToDelete || isAdmin}
+						isDisabled={!hasPermissionToDelete || isSameUser}
+						isLoading={isDeleting}
 						label={UsersTableHeader.BUTTONS}
+						loaderColor="orange"
+						onClick={handleOpenDeleteModal(user)}
```

Comment:
```suggestion
						onClick={handleOpenDeleteModal}
```
we do not want to have currying functions